aws/cloudformation/drone-stack.yml

```
@@ -36,7 +36,7 @@ Parameters:
   WorkerInstanceType:
     Type: String
     Description: EC2 Instance Type that Autoscaler provisions to run Builds.
-    Default: m5.xlarge
+    Default: m5.4xlarge
```

For consistency [with our production servers](https://github.com/code-dot-org/code-dot-org/blob/905525f78a1b1c4c75badb2a46ef1e54019a1217/aws/cloudformation/cloud_formation_stack.yml.erb#L62), could we use `m7i`?
```suggestion
    Default: m7i.4xlarge
```